src/query/service/src/interpreters/access/privilege_access.rs

```diff
@@ -952,6 +952,30 @@ impl AccessChecker for PrivilegeAccess {
                 self.validate_access(&GrantObject::Global, UserPrivilegeType::Super)
                     .await?;
             }
+            Plan::ShowGrants(plan) => {
+                let current_user = self.ctx.get_current_user()?;
+                if let Some(principal) = &plan.principal {
+                   match principal {
+                       PrincipalIdentity::User(user) => {if current_user.identity() == *user {
```

Comment:
What's the code style here?

Reply:
Strange, I run cargo fmt on my local.

Reply:
Delete the {} in the newset commit.

Reply:
Not related to `{}`, my RustRover works fine:
<img width="792" alt="image" src="https://github.com/datafuselabs/databend/assets/172204/a424d7b9-297e-4859-9f79-56d264aae7a2">